packages/nodes-base/nodes/WhatsApp/WhatsAppTrigger.node.ts

```diff
@@ -112,7 +81,99 @@ export class WhatsAppTrigger implements INodeType {
 				type: 'notice',
 				default: '',
 			},
-			...whatsappTriggerDescription,
+			{
+				displayName: 'Trigger On',
+				name: 'updates',
+				type: 'multiOptions',
+				required: true,
+				default: [],
+				options: [
+					{
+						name: 'Account Review Update',
+						value: 'account_review_update',
+					},
+					{
+						name: 'Account Update',
+						value: 'account_update',
+					},
+					{
+						name: 'Business Capability Update',
+						value: 'business_capability_update',
+					},
+					{
+						name: 'Message Template Quality Update',
+						value: 'message_template_quality_update',
+					},
+					{
+						name: 'Message Template Status Update',
+						value: 'message_template_status_update',
+					},
+					{
+						name: 'Messages',
+						value: 'messages',
+					},
+					{
+						name: 'Phone Number Name Update',
+						value: 'phone_number_name_update',
+					},
+					{
+						name: 'Phone Number Quality Update',
+						value: 'phone_number_quality_update',
+					},
+					{
+						name: 'Security',
+						value: 'security',
+					},
+					{
+						name: 'Template Category Update',
+						value: 'template_category_update',
+					},
+				],
+			},
+			{
+				displayName: 'Options',
+				name: 'options',
+				type: 'collection',
+				default: {},
+				placeholder: 'Add option',
+				options: [
+					{
+						// https://developers.facebook.com/docs/whatsapp/cloud-api/webhooks/payload-examples#message-status-updates
+						displayName: 'Receive Message Status Updates',
+						name: 'messageStatusUpdates',
+						type: 'multiOptions',
+						default: ['all'],
+						description:
+							'The WhatsApp sends notifications to inform you of the status of the messages. Select the types of notifications you want to receive.',
```

Comment:
I think wording here could be better, maybe sync with DavidA or Jake on the copy of this new option